comp/forwarder/defaultforwarder/blocked_endpoints.go

```diff
@@ -67,11 +80,29 @@ func (e *blockedEndpoints) close(endpoint string) {
 	if knownBlock, ok := e.errorPerEndpoint[endpoint]; ok {
 		b = knownBlock
 	} else {
-		b = &block{}
+		b = &block{
+			state: Closed,
+		}
 	}
 
-	b.nbError = e.backoffPolicy.IncError(b.nbError)
-	b.until = time.Now().Add(e.getBackoffDuration(b.nbError))
+	switch b.state {
+	case Closed:
+		// The circuit breaker is closed, we need to open it for a given time
+		b.nbError = e.backoffPolicy.IncError(b.nbError)
+		b.until = TimeNow().Add(e.getBackoffDuration(b.nbError))
+		b.state = Open
```

Comment:
Would it make sense to align terminology between circuit breakers states and the API? `Open` is when the transactions are blocked, but the method to transition into this state is called `close`.